python/src/trezorlib/transport/hid.py

```diff
@@ -132,11 +133,17 @@ def get_path(self) -> str:
         return f"{self.PATH_PREFIX}:{self.device['path'].decode()}"
 
     @classmethod
-    def enumerate(cls, debug: bool = False) -> Iterable["HidTransport"]:
+    def enumerate(
+        cls, models: Optional[Iterable["TrezorModel"]] = None, debug: bool = False
```

Comment:
Isn't it better to use the following?
```suggestion
        cls, models: Iterable["TrezorModel"] = {TREZOR_ONE}, debug: bool = False
```
and drop the `if models is None: models = ...` below?

Reply:
The problem with this is that passthrough from `trezorlib.transport.enumerate()` will not work. The general `enumerate()` method doesn't know which models are appropriate for the specific transports, so it will pass in `None`.

Reply:
Okay, thanks